docs/tutorials/federated_learning_for_image_classification.ipynb

```diff
@@ -82,7 +82,7 @@
         "This tutorial, and the Federated Learning API, are intended primarly for users\n",
         "who want to plug their own TensorFlow models into TFF, treating the latter\n",
         "mostly as a black box. For a more in-depth understanding of TFF and how to\n",
-        "implement your own federated learning algorithms, see the tutorials on the FC Core API - [Custom Federated Algorithms Part 1](custom_federated_algorithms_1.ipynb) and [Part 2](custom_federated_algorithms_2.ipynb).\n",
+        "implement your own federated learning algorithms, see the tutorials on the FC Core API - [Custom Federated Algorithms Part 1](https://github.com/tensorflow/federated/blob/master/docs/tutorials/custom_federated_algorithms_1.ipynb) and [Part 2](https://github.com/tensorflow/federated/blob/master/docs/tutorials/custom_federated_algorithms_2.ipynb).\n",
```

Comment:
Can you update this change to use the following links instead?

`https://github.com/tensorflow/federated/blob/v0.2.0/docs/tutorials/custom_federated_algorithms_1.ipynb`

Basically, we want the colabs to point at the latest release version, not master.

Reply:
@michaelreneer : Thanks for your feedback, will certainly do the changes and upload, Thanks!